shanoir-ng-front/src/app/shared/components/table/table.component.html

```diff
@@ -33,73 +33,78 @@
                     [style.background-color]="vars.cellGraphics?.tag ? null : vars.cellGraphics?.backgroundColor"
                     #refTd
                     >
+                    <div class="cell-container" [class.table-edit]="editMode" *ngIf="col.type != 'button'" [class.select]="!!col.possibleValues" [class.multiselect]="!!col.possibleValues && col.multi" [class.wrap]="!col.nowrap">
```

Comment:
indentation : not important at all but since you have other fixes here the div should be one tab back